src/main/java/com/stories/service/StoriesServiceImpl.java

```diff
@@ -116,6 +123,20 @@ public StoryDomain getStoryById(String id) throws Exception {
 		logger.debug("Getting all stories - JSON : {}", allStoriesDomain);
 		return allStoriesDomain;
 	}
+	
+	public List<TasksDomain> findTasksByStoryId(String id) throws EntityNotFoundException{
+		if(storiesRepository.existsById(id)) {
+			Aggregation aggregation = Aggregation.newAggregation(
```

Comment:
Please lest follow the standars ,this Aggretion objet should be on mongo Repository

Reply:
fixed